### ISSUE TITLE
Example for multi-level grid

### ISSUE THREAD

**Issue by Haider-BA** [luanmingyi/ordered_tec#3]
Kindly add an example to get PLT for multi-grid/multi-domain (refined grids) and corresponding flow variables. Also, adding geometries ( with time).

**Comment by luanmingyi** [comment]
I will give some example later, and you need to read some document of tecplot. Sometimes it is the question about tecplot not about the library.

**Comment by luanmingyi** [comment]
And for the geometries, maybe i will add it to the library, but the geometries with time is not supported by tecplot.

**Comment by Haider-BA** [comment]
Yes, I have agreed with you that sometimes it is TecPlot documentation and also geometries with time are not supported. Problem is that I would like to multi-zones (known at compile time). Writing one zone or even two with the library is easy but when writing more zones and their corresponding variables, it is quite difficult. I think this library will solve my problem. I request to give an example to write zones recursively and without recursion. 
Thank you for your help.
